doc/api/dgram.md

```diff
@@ -283,6 +283,9 @@ not work because the packet will get silently dropped without informing the
 source that the data did not reach its intended recipient.
 
 ### socket.setBroadcast(flag)
+<!-- YAML
+added: v0.6.9
```

Comment:
Hm, I suppose it's possible I wrote the libuv code but forgot to hook it up in node... let's pretend this conversation never happened.